postgrest/base_request_builder.py

```diff
@@ -366,30 +366,45 @@ def __init__(
         BaseFilterRequestBuilder.__init__(self, session, headers, params)
 
     def order(
-        self: _FilterT, column: str, *, desc: bool = False, nullsfirst: bool = False
+        self: _FilterT,
+        column: str,
+        *,
+        desc: bool = False,
+        nullsfirst: bool = False,
+        foreign_table: str = "",
     ) -> _FilterT:
         """Sort the returned rows in some specific order.
 
         Args:
             column: The column to order by
             desc: Whether the rows should be ordered in descending order or not.
-            nullsfirst: nullsfirst
+            nullsfirst: nullsfirst,
+            foreign_table: Order results in foreign table
         """
         self.params = self.params.add(
-            "order",
+            f"{foreign_table + '.order' if foreign_table else 'order' }",
```

Comment:
This f-string expression is very hard to read
```suggestion
            f"{foreign_table}.order" if foreign_table else "order" ,
``` 
What do you think about writing it like this?